src/status_im/contexts/wallet/wallet_connect/core.cljs

```diff
@@ -0,0 +1,27 @@
+(ns status-im.contexts.wallet.wallet-connect.core
+  (:require [status-im.constants :as constants]
+            [utils.transforms :as transforms]))
+
+(defn extract-native-call-signature
+  [data]
+  (-> data transforms/json->clj :result))
+
+(defn chain-id->eip155
+  [chain-id]
+  (str "eip155:" chain-id))
+
+(defn format-eip155-address
+  [address chain-id]
+  (str chain-id ":" address))
+
+(defn event->method
```

Comment:
Normally in Clojure I think we wouldn't name using the `->` arrow convention because we're not transforming anything. The function is an extractor, so it could be named `get-request-method` for instance.